migrate/20231130_postgres_earliest_backup.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:postgres_timeline) do
+      add_column :earliest_backup_at, :timestamptz
```

Comment:
Can we rename this to earliest_backup_completed_at?

The reason is that I'm adding latest_backup_started_at and latest_backup_completed_at fields. earliest_backup_completed_at would suit more with those column names.